views/user_sessions/new.haml

```diff
@@ -0,0 +1,9 @@
+.login-form
+  %form.form-signin{action: '/user_sessions/create', method: 'POST'}
+    =Rack::Csrf.tag(env)
+    %h2.form-signin-heading Please sign in
+    %label.sr-only{:for => "email"} Email address
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace